testing/python/transform/test_tilelang_transform_inject_assumes.py

```
@@ -0,0 +1,44 @@
+import tilelang.language as T
+import tilelang.testing
+
+
+def _run_inject_assumes(func: tvm.tirx.PrimFunc):
+    mod = tvm.IRModule.from_expr(func.with_attr("global_symbol", "main"))
+    return tl.transform.InjectAssumes()(mod)
+
+
+def test_inject_assumes_warns_on_local_var_assume(capfd):
+    @T.prim_func
+    def main(A: T.Tensor((128,), T.float32), l: T.int32):
```

Ruff (0.15.15)

[error] Ambiguous variable name: `l`

(E741)